pkg/cmd/run.go

```diff
@@ -97,6 +98,7 @@ func newCmdRun(rootCmdOptions *RootCmdOptions) (*cobra.Command, *runCmdOptions)
 	cmd.Flags().StringArray("property-file", nil, "Bind a property file to the integration. E.g. \"--property-file integration.properties\"")
 	cmd.Flags().StringArray("label", nil, "Add a label to the integration. E.g. \"--label my.company=hello\"")
 	cmd.Flags().StringArray("source", nil, "Add source file to your integration, this is added to the list of files listed as arguments of the command")
+	cmd.Flags().String("pod-template", "", "Custom PodTemplateSpec that will be used in integrations")
```

Comment:
I'd suggest "The path of the YAML file containing a PodSpec template to be used for the Integration pods".